### DISCUSSION TITLE
How do I remove the managed users?

### DISCUSSION THREAD

**Discussion by Tierney300** [community/community#38617] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.88.0 (82868297)
Device: iPhone SE 3 (iOS 16.0.3)
</details>

**Comment by david-engelmann** [comment]
I use SSH, the credential helper is also an option but I find it to be not as seamless as SSH Authentication

**Reply by Tierney300** [reply]
What is a credential helper?
My iPhone gets zoom bombers, fake accounts have been created using my name and photo.
